lib/div-alg.gi

```diff
@@ -403,13 +403,21 @@ S[1]:=g1;
 
 if Length(A)=2 then d:=1; fi;
 if Length(A)=4 then d:=A[4][1]; F1:=NF(A[3],[A[4][2]]); fi;
-if (Length(A)=5 and Length(A[4])=2) then
-   d:=A[4][1][1]*A[4][2][1];
-   F1:=NF(A[3],[A[4][1][2],A[4][2][2]]);
-fi;
-if (Length(A)=5 and Length(A[4])=3) then
-   d:=A[4][1][1]*A[4][2][1]*A[4][3][1];
-   F1:=NF(A[3],[A[4][1][2],A[4][2][2],A[4][3][2]]);
+#if (Length(A)=5 and Length(A[4])=2) then
```

Comment:
Do we still need this code? Perhaps we can remove it, it will still be in revisions history? Or maybe leave a comment explaining why it has been commented out, why kept for the future, for now long etc.

Reply:
No this can go.  I think we will eventually clean out all of the commented lines in div-alg.  For now leave them in case the tests have problems we can see what we were doing before.